src/main/kotlin/com/terraformation/backend/tracking/model/ObservationResultsModel.kt

```diff
@@ -423,3 +453,18 @@ fun List<ObservationSpeciesResultsModel>.unionSpecies(
         )
       }
 }
+
+/** Calculate standard deviation */
+fun Collection<Int>.calculateStandardDeviation(): Int? {
+  if (this.size <= 1) {
+    return null
+  }
+
+  val numSamples = this.size.toDouble()
+  val samples = this.map { it.toDouble() }
+  val mean = samples.sumOf { it } / numSamples
```

Comment:
Could use `samples.sum()` instead of `sumOf { it }`.

Reply:
I should just use `.average()`. I originally used `BigDecimal` which doesn't have an average function. They also don't have a sqrt function which is why I am using Double.